modules/PointReactorKinetics/point_reactor_kinetics.h

```diff
@@ -38,6 +36,8 @@ class TransientSolver : public chi_physics::Solver
   void Step() override;
   void Advance() override;
 
+  chi::ParameterBlock GetInfo(const chi::ParameterBlock& params) const override;
+
   // Getters and Setters
   double PopulationPrev() const;
   double PopulationNext() const;
```

Comment:
Irrelevant to the PR, but I would suggest using `PopulationPrev` and `PopulationNew` to be more consistent with solvers. I thought `Next` meant to take a time step or something on first glance.

Reply:
Makes sense. Changed to new.